tests/functional/warehouse_test/test_warehouses.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from dbt.tests.util import (
+    run_dbt,
+    check_relations_equal
+)
+from tests.functional.warehouse_test.fixtures import (
+    models,
+    project_config_models,
+    project_files
+    ) # noqa: F401
+import os
+
+
+class TestModelWarehouse():
+    def dir(value):
+        return os.path.normpath(value)
+
+    @pytest.fixture(scope="class")
+    def model_path(self):
+        return self.dir('models')
```

Comment:
Is this pattern something that needs to be done for snowflake?  It's just not a pattern I've seen in our tests yet but I haven't worked with the snowflake tests either.